core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1475,6 +1475,24 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey TOTAL_EXTRA_TIME =
+       new Builder("JvmPauseMonitor.totalExtraTime")
+           .setDescription("Total extra sleep time.")
```

Comment:
can you provide more detailed metrics descriptions for all three new metric key?
note that those description may be viewed by users who don't know what jvm pause monitor is.
better to change the metric name to be more user friendly instead of developer friendly, thanks!